modules/dataquery/help/dataquery.md

```diff
@@ -0,0 +1,54 @@
+# Data Query Tool (DQT)
+
+This module provides a simple and easy-to-use interface for querying and exporting data from LORIS.
+
+There are several tabs within this module, including the default **Info** tab, which provides explanations for the other tabs.
+
+## Define Fields
+
+This tab allows you to select the data categories and fields to add to your query. You can add fields via the Instrument dropdown menu, the *Search within Instrument* search bar, or the *Add All* function. 
+Note: once you select an instrument, its field names and descriptions will appear in a table for you to select from. You can also specify visits. 
+
+You can also use the *Load Saved Query* menu to select a previously saved query, rather than generating a new set of fields and filters to query from scratch. 
+
+Once you have your fields defined, you can continue on to the **Define Filters** tab, or directly to the **View Data** tab if your query does not require filters. 
+
+## Define Filters
+
+This is where you define the criteria to filter the data for your query. If you have You can apply filters on any field in order to narrow the set of data records returned by a query, such as filtering for a specific site or visit.
+
+You can apply multiple filters, and layer them with "And/Or" conditional logic. Filters are applied at the level of the candidate. The data fields are grouped by category.
+
+To add a filter, select an instrument using the dropdown. A secondary dropdown will appear, listing all data fields in that category. Once a data field is selected, additional dropdowns to specify the “Operator” and “Value” will appear. These _Operators_ are:
+
+   = equal to<br>
+   != does not equal<br>
+   <= less than or equal to<br>
+   \>= greater than or equal to<br>
+   _startsWith_ : must begin with a specific character or string<br>
+   _contains_ : must contain a specific character or string<br>
+
```

Comment:
Missing info for `isNull` and `isNotNull`.

Reply:
ok ill make a commit.

Reply:
ready for re-review @PapillonMcGill